apps/api-gateway/src/connection/connection.service.ts

```diff
@@ -95,9 +95,12 @@ export class ConnectionService extends BaseService {
     return this.sendNatsMessage(this.connectionServiceProxy, 'receive-invitation', payload);
   }
 
-  async _getWebhookUrl(tenantId: string): Promise<string> {
+  async _getWebhookUrl(tenantId?: string, orgId?: string): Promise<string> {
     const pattern = { cmd: 'get-webhookurl' };
-    const payload = { tenantId };
+
+    const getWebhook = { tenantId, orgId };
+    
+    const payload = { getWebhook };
```

Comment:
```suggestion
    const payload = { tenantId, orgId};
```

Reply:
This function is calling the webhook controller, and in the webhook service controller, we use it in that format while listening. That's why I’ve bound the tenantId and orgId to the payload and sent it